org-code-javabuilder/lang/src/main/java/org/code/lang/System.java

```diff
@@ -0,0 +1,29 @@
+package org.code.lang;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+
+/**
+ * User-facing System class. We only support a few methods and fields from java.lang.System, so we
+ * wrap java.lang.System in org.code.lang.System.
+ */
+public class System {
+    public static final InputStream in = java.lang.System.in;
+    public static final PrintStream out = java.lang.System.out;
+
+    public static void arraycopy(Object src, int srcPos, Object dest, int destPos, int length) {
```

Comment:
Nit: `private System() {}` constructor? That's what it looks like is in real System. Not that there's anything you could do with a system object, but feels like something a student who is learning might try / might be good to be consistent with java.

Reply:
good catch, added